src/infrastructure/utils/screenshot.ts

```diff
@@ -0,0 +1,24 @@
+import html2canvas from 'html2canvas';
+
+/**
+ * Make html element screenshot
+ * @param elementId - id of element, which have to be screenshooted
+ * @returns binary image data
+ */
+export async function makeElementScreenshot(elementId: string): Promise<Blob | null> {
```

Comment:
its better to pass element to not to care about id assigning

Reply:
move a lot of logic to util